pages/who-we-are-hiring.md

```diff
@@ -20,10 +20,10 @@ Everywhere in the United States! If you have no experience working on remote tea
 We're actively hiring for the following positions:
 
 ###Software development (front end)
-Using modern, open source web frameworks and tools, our front and back end developers work in small, cross-functional teams, close to user research and product prioritization, to produce functional, scalable web services.
```

Comment:
@fureigh The edit was for readability. The original sentence had too many commas and clauses to be easily readable, and we should avoid jargony words like prioritization. If it's important to mention how the team relies on user research and has a hand in product prioritization, I think that should be moved to another sentence.